tsconfig.esm.json

```diff
@@ -1,9 +1,6 @@
 {
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
-    "outDir": "./build/esm",
-    "module": "ESNext",
-    "target": "ES2017",
```

Comment:
Same here, both these fields are already set in the parent config, so we can omit them here.